examples/python/hackathon_app/README.md

```diff
@@ -87,6 +87,89 @@ yarn start
 
 - this launches the frontend static server at http://127.0.0.1:3000 which proxies fetch requests to the backend server
 
+## Production deployment
+
+Deployment is partially automated. devops has an ansible runbook that they execute manually when we want a new version deployed.
+It currently looks something like this (I don't have access to the repo where it lives so this is probably stale but you'll get the idea)
+```
+- name: Docker image and push
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  # currently we have these secrets/vars stored:
+  #   export LOOKERSDK_CLIENT_ID={{ lookersdk_client_id }}
+  #   export LOOKERSDK_CLIENT_SECRET={{ lookeradk_client_secret }}
+  #   export GOOGLE_SHEET_ID={{ google_sheet_id }}
+  #   export GOOGLE_APPLICATION_CREDENTIAL_ENCODED={{ google_application_credential_encoded }}  # currently using credentials from a service account in the looker-se GCP project: https://console.cloud.google.com/home/dashboard?project=looker-se
+  #   export FLASK_WTF_CSRF_SECRET_KEY={{ flask_wtf_csrf_secret_key }}
+  #   export FLASK_SECRET_KEY={{ flask_secret_key }}
+  #   export LOOKERSDK_API_VERSION='3.1'
+  #   export LOOKERSDK_BASE_URL='https://hack.looker.com:19999'
+  #   export GOOGLE_APPLICATION_CREDENTIALS='./google-creds.json'
+  vars_files:
+    - roles/dockerapps/vars/hackathonapp_vault.yml
+    - roles/dockerapps/vars/hackathonapp_vars.yml
+
+  tasks:
+
+      # checkout hackathonapp repo
+      - git:
+          repo: git@github.com:looker-open-source/sdk-examples.git
+          dest: "{{hackathonapp_checkout_location}}"
+          version: "{{hackathonapp_app_version}}"
+        register: git_info
+
+      # write .env with info
+      - template: src=roles/dockerapps/templates/hackathonapp_.env.j2 dest="{{ hackathonapp_app_location }}/env.list"
+
+      # write status.json with info
+      - template: src=roles/dockerapps/templates/hackathonapp_status.json.j2 dest="{{ hackathonapp_checkout_location }}/status.json"
```

Comment:
Ah yes, you said to add a status function to the service. I need to add 2 functions.